lib/pa11y.js

```diff
@@ -87,7 +87,11 @@ async function runPa11yTest(url, options, state) {
 	// We use a state object which is accessible from the
 	// wrapping function
 	options.log.debug('Launching Headless Chrome');
-	const browser = state.browser = await puppeteer.launch(options.chromeLaunchConfig);
+
+	const puppeteerConfig = options.chromeLaunchConfig;
+	puppeteerConfig.timeout = options.timeout;
+
+	const browser = state.browser = await puppeteer.launch(puppeteerConfig);
```

Comment:
It feels a little bit odd to me to use the same timeout option to both set the Headless Chrome startup timeout as well as the page timeout.

I suggest we remove this part, so if someone wants to set a Chrome startup timeout they can still do that separately within the options.chromeLaunchConfig variable without it getting overridden by the page-specific loading (`goto`) timeout.

Reply:
Thanks for review @andrewmee! 

Yeah, that wasn't an easy decision. When I had taken this decision I was thinking that could be redundant for users need to pass two timeouts to pa11y. In a slow page case, I think that will be necessary to set both timeouts for effectively works.

Anyway, I'll change accordingly with your suggestion as soon as possible :)

Reply:
I've made the changes that you suggest.

Also I've understood that puppeteer timeout is only related to chromium launch, as I found in puppeteer docs:

`timeout <number> Maximum time in milliseconds to wait for the Chrome instance to start. Defaults to 30000 (30 seconds). Pass 0 to disable timeout.` 

Thx